eisbuk-admin/src/App.tsx

```diff
@@ -37,4 +40,22 @@ const App: React.FC = () => {
   );
 };
 
+// ***** Region Styles ***** //
+const useStyles = makeStyles((theme) => ({
+  root: {
+    margin: theme.spacing(0.75, 0),
+
+    /**
+     * This is, hopefully, temporary workaround to a
+     * bug in notistack that causes snackbars to be unclickable.
+     * Notistack v.1.0.10 fixes that bug, however,
+     * it only works with materialUI v^5, which is in pre-release currently.
+     * The bug is caused by the root class not having the 'pointerEvents'
+     * property set to 'all', 'root' class here overrides that.
+     * @TODO upgade MUI, notistack and remove this when MUI v5 becomes the LTS
+     */
+    pointerEvents: "all",
```

Comment:
It's always best to write comments above (my preference) or inline with the part we're refering to. 
*fixed: the fix only works with MUI v^5, add `@TODO` flag*